src/ring_theory/ideal/operations.lean

```diff
@@ -1280,6 +1280,44 @@ variables [comm_ring R] [comm_ring S]
 @[simp] lemma mk_ker {I : ideal R} : (quotient.mk I).ker = I :=
 by ext; rw [ring_hom.ker, mem_comap, submodule.mem_bot, quotient.eq_zero_iff_mem]
 
+lemma map_mk_eq_bot_of_le {I J : ideal R} (h : I ≤ J) : I.map (J^.quotient.mk) = ⊥ :=
+by {rw [map_eq_bot_iff_le_ker, mk_ker], exact h}
+
+lemma ker_quotient_lift {S : Type v} [comm_ring S] {I : ideal R} (f : R →+* S) (H : I ≤ f.ker) :
+  (ideal.quotient.lift I f H).ker = (f.ker).map I^.quotient.mk :=
+begin
+  ext x,
+  split,
+ {intro hx,
```

Comment:
```suggestion
  { intro hx,
```
Please indent the rest of the block, and add a space before the closing `}`.

Other blocks also need such reformatting.